openssl/src/asn1.rs

```diff
@@ -88,10 +154,12 @@ impl Asn1StringRef {
         }
     }
 
+    /// Return the string as an array of bytes
```

Comment:
Can we note here that the bytes may not be in a standard UTF8 compatible encoding and point to as_utf8?